services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java

```diff
@@ -243,7 +243,7 @@ private void onTick(Date date) {
           this.currentEpochGauge.set(compute_epoch_at_slot(nodeSlot).longValue());
           STDOUT.log(Level.INFO, "******* Slot Event *******", ALogger.Color.WHITE);
           STDOUT.log(Level.INFO, "Node slot:                             " + nodeSlot);
-          Thread.sleep(SECONDS_PER_SLOT * 1000 / 2);
+          Thread.sleep(SECONDS_PER_SLOT * 1000 / 3);
```

Comment:
Not for this bit of work, but I've logged https://pegasys1.atlassian.net/browse/BC-101 to replace this type of `Thread.sleep` call with a more event driven timer.  Only really matter down the track when we start thinking through how to catch up if we fell behind.

Reply:
Agreed.